include/highfive/bits/H5Converter_misc.hpp

```diff
@@ -625,7 +625,18 @@ struct inspector<Eigen::Matrix<T, M, N>> {
     static constexpr bool is_trivially_copyable = std::is_trivially_copyable<value_type>::value &&
                                                   inspector<value_type>::is_trivially_copyable;
 
+
+    static void assert_not_buggy(Eigen::Index nrows, Eigen::Index ncols) {
+        if (nrows > 1 && ncols > 1) {
+            throw std::runtime_error(
+                "HighFive has been broken for Eigen::Matrix. Please check "
+                "https://github.com/BlueBrain/HighFive/issues/532.");
```

Comment:
```suggestion
                "HighFive has been broken for Eigen::Matrix. "
                "Please use H5Easy or open a PR. See "
                "https://github.com/BlueBrain/HighFive/issues/532.");
```

Reply:
It's unfortunately not that simple. Just switching to H5Easy isn't going to fix the issue of existing files. The point of breaking this is to make people check the ticket so that they understand how they're affected by this issue.

Reply:
You could say "Please use H5Easy to write NEW data. Note that old data may be 'corrupted'"

Reply:
I'm not sure I understand what you want to achieve by suggestion H5Easy as an alternative, it's not a drop in replacement (Vectors serialized from core are 2D; while Easy strips the singleton dimension).

I would like the message to read that they need to go read a full length piece of text; and then figure out how this affects them.

The point is to end the indecision.

Reply:
It's fine, you can keep what you had.